src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random(100);
```

Comment:
let's use constant for new Random
```suggestion
        Random random = new Random(100);
```